congestion-control-charter.txt

```diff
@@ -51,6 +51,9 @@ requires approval of the Responsible AD.
 on an assessment of their maturity and likelihood of near-term wide-scale
 deployment.
 
+* Multipath congestion control that detects when two paths share the same
+bottleneck.

```

Comment:
I might be missing context here, but: Can we be more generic here and simply say "Multipath congestion control"? I'm not sure if the detection in a mandatory component of such an algorithm, but also I'm happy to be corrected.

Reply:
I think that's the actual problem to solve, but if that explanatory text troubles you I can delete it.